spec/features/user_downvotes_on_review_spec.rb

```diff
@@ -7,37 +7,34 @@
 } do
 
   describe 'User downvotes on a review' do
-    scenario 'user clicks downvote' do
+    scenario 'user clicks downvote', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.